server/supernova/sc/sc_ugen_factory.hpp

```diff
@@ -67,7 +67,8 @@ class sc_ugen_def : public aligned_class, public named_hash_entry {
         flags(inFlags),
         unitcmd_set(unitcmd_set_type::bucket_traits(unitcmd_set_buckets, unitcmd_set_bucket_count)) {}
 
-    Unit* construct(sc_synthdef::unit_spec_t const& unit_spec, sc_synth* s, World* world, linear_allocator& allocator);
+    Unit* construct(sc_synthdef::unit_spec_t const& unit_spec, sc_synth* synth, int synthIndex, World* world,
```

Comment:
parameter names differ between declaration and definition (synth[Index] vs parent[Index])